platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Organization.java

```diff
@@ -15,7 +16,7 @@
 @Table(name = "organization",
         uniqueConstraints = {@UniqueConstraint(name = "nick", columnNames = "nick")}
 )
-public class Organization extends IdentifiableEntity implements CanonicalEntity {
+public class Organization extends IdentifiableEntity implements CanonicalEntity, Serializable {
```

Comment:
Class now implements `Serializable` but is missing field `serialVersionUID` which is strongly recommended.